pandas/core/generic.py

```diff
@@ -4260,16 +4271,76 @@ def _get_values(self):
         return self.values
 
     def get_values(self):
-        """same as values (but handles sparseness conversions)"""
+        """
+        Same as values (but handles sparseness conversions).
+
+        Returns
+        -------
+        numpy.ndaray
+            Numpy representation of NDFrame
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'a': np.random.randn(2).astype('f4'),
+        ...                    'b': [True, False], 'c': [1.0, 2.0]})
+        >>> df.get_values()
+        array([[0.25209328532218933, True, 1.0],
+        [0.35383567214012146, False, 2.0]], dtype=object)
```

Comment:
I don't think this formatting is quite right. Can you try using non-random data and running the doctest on it?